chapter2/tests/t20/terraform/base/alb.tf

```diff
@@ -0,0 +1,46 @@
+module "alb" {
+  source             = "terraform-aws-modules/alb/aws"
+  version            = "8.7.0"
+  name               = local.labels.alb
+  load_balancer_type = "application"
+  vpc_id             = data.aws_vpc.target.id
+  subnets            = data.aws_subnets.wordpress.ids
+  security_groups    = [module.wordpress_alb_sg.security_group_id]
+  target_groups = [
+    {
+      backend_port     = 80
+      backend_protocol = "http"
+      name             = local.labels.tg
+      target_type      = "instance"
+      health_check = {
+        enabled             = true
+        interval            = 30
+        path                = "/"
+        port                = "traffic-port"
+        healthy_threshold   = 5
+        unhealthy_threshold = 2
+        timeout             = 6
+        protocol            = "HTTP"
+        matcher             = "200-399"
+      }
+      targets = [for i in range(length(module.wordpress_ec2_instance)) : {
```

Comment:
I suppose you can do this like this and it will look simpler (because your `module.wordpress_ec2_instance` will be list of objects):

```
targets = [for item in module.wordpress_ec2_instance : {
        target_id =item.id
        port      = 80
 }]
```

Reply:
> I suppose you can do this like this and it will look simpler (because your `module.wordpress_ec2_instance` will be list of objects):

@kseniyashaydurova 
Thanks, changed